packages/@uppy/svelte/package.json

```diff
@@ -3,9 +3,33 @@
   "version": "4.5.0",
   "description": "Uppy plugin that helps integrate Uppy into your Svelte project.",
   "type": "module",
-  "svelte": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "main": "./dist/index.js",
+  "sideEffects": [
+    "*.css"
+  ],
+  "files": [
+    "dist",
+    "src"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js"
+    },
+    "./css/style.css": "./dist/styles.css",
+    "./dashboard": {
+      "types": "./dist/components/Dashboard.svelte.d.ts",
+      "svelte": "./dist/components/Dashboard.svelte"
+    },
+    "./dashboard-modal": {
+      "types": "./dist/components/DashboardModal.svelte.d.ts",
+      "svelte": "./dist/components/DashboardModal.svelte"
+    },
+    "./status-bar": {
+      "types": "./dist/components/StatusBar.svelte.d.ts",
+      "svelte": "./dist/components/StatusBar.svelte"
+    },
```

Comment:
Why can't we simply export everything from index? Also status bar is gone?

Reply:
yes we can do it in case of svelte since it does only has dashboard as it's peer dep.

Reply:
let's do this after merging status-bar PR ?